S09-hashes/objecthash.t

```diff
@@ -140,6 +140,11 @@ plan 55;
     eval-lives-ok 'my %*a{Int}', "Accept dynamic object hash"
 }
 
+# https://github.com/rakudo/rakudo/issues/1486
+{
+    ok {:foo, (if 1 { :bar })}, 'a conditional inside circumfix {} works';
```

Comment:
Are we sure the static optimizer doesn't optimize the `if 1` away?

Reply:
I hope the new version is complex enough?